MAUI/Button/Getting-Started.md

```diff
@@ -153,3 +153,6 @@ button.WidthRequest = 150;
 ![Button with background image](images/getting-started/ButtonWithBackgroundImage.png)
 
 Find the complete getting started sample of the .NET MAUI Button from this [link.](https://github.com/SyncfusionExamples/maui-button-samples)
+
+N> You can also explore our [.NET MAUI Button Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/Buttons) that shows you how to render the Button in .NET MAUI.
```

Comment:
Use the below format
N> You can refer to our [.NET MAUI Tab View](https://www.syncfusion.com/maui-controls/maui-tab-view) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Tab View Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/TabView) that shows you how to render the Tab View in .NET MAUI.

Reply:
Resolved